api/src/pcapi/routes/backoffice/collective_offers/collective_offers_blueprint.py

```diff
@@ -287,8 +307,8 @@ def _get_collective_offer_ids_query(form: forms.GetCollectiveOfferAdvancedSearch
 
 
 def _get_collective_offers(
-    form: forms.GetCollectiveOfferAdvancedSearchForm,
-) -> list[educational_models.CollectiveOffer]:
+    collective_offers_ids: list[int] | sa.orm.Query,
+) -> sa.orm.Query:
```

Comment:
Tu dois même pouvoir typer explicitement : 
```suggestion
) -> sa.orm.Query[educational_models.CollectiveOffer]:
```

Reply:
c'est l'inverse, on en recharge aucune (puisqu'on a rien touché il est inutile de recharger des offres)

Reply:
j'avais deja essayé malheureusement ça ne marche pas 
`TypeError: type 'Query' is not subscriptable`